morf-postgresql/src/main/java/org/alfasoftware/morf/jdbc/postgresql/PostgreSQLDialect.java

```diff
@@ -93,6 +94,25 @@ protected String schemaNamePrefix(Table table) {
   }
 
 
+  @Override
+  protected String getSqlForRowNumber(){
+    return "ROW_NUMBER() OVER()";
+  }
+
+
+  @Override
+  protected String getSqlForWindowFunction(Function function) {
+    FunctionType functionType = function.getType();
+    switch (functionType) {
+      case ROW_NUMBER:
+        return "ROW_NUMBER()";
+
+      default:
+        return getSqlFrom(function);

```

Comment:
We should call `super.getSqlForWindowFunction(function)`, which will call `getSqlFrom(function)` for us.